### ISSUE TITLE
Library fails to install

### ISSUE THREAD

**Issue by thanatos** [richardhenry/hotqueue#21]
Currently, hotqueue appears to fail to install:

```
$ env/bin/pip install hotqueue
Collecting hotqueue
  Using cached hotqueue-0.2.7.tar.gz
    Complete output from command python setup.py egg_info:
    Downloading http://pypi.python.org/packages/source/d/distribute/distribute-0.6.14.tar.gz
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-RJ3enJ/hotqueue/setup.py", line 7, in <module>
        from distribute_setup import use_setuptools; use_setuptools()
      File "/tmp/pip-build-RJ3enJ/hotqueue/distribute_setup.py", line 145, in use_setuptools
        return _do_download(version, download_base, to_dir, download_delay)
      File "/tmp/pip-build-RJ3enJ/hotqueue/distribute_setup.py", line 124, in _do_download
        to_dir, download_delay)
      File "/tmp/pip-build-RJ3enJ/hotqueue/distribute_setup.py", line 193, in download_setuptools
        src = urlopen(url)
      File "/usr/lib/python2.7/urllib2.py", line 154, in urlopen
        return opener.open(url, data, timeout)
      File "/usr/lib/python2.7/urllib2.py", line 435, in open
        response = meth(req, response)
      File "/usr/lib/python2.7/urllib2.py", line 548, in http_response
        'http', request, response, code, msg, hdrs)
      File "/usr/lib/python2.7/urllib2.py", line 473, in error
        return self._call_chain(*args)
      File "/usr/lib/python2.7/urllib2.py", line 407, in _call_chain
        result = func(*args)
      File "/usr/lib/python2.7/urllib2.py", line 556, in http_error_default
        raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
    urllib2.HTTPError: HTTP Error 403: SSL is required

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-RJ3enJ/hotqueue/
```

I think this is due to a [recent change](https://mail.python.org/pipermail/distutils-sig/2017-October/031712.html) that PyPI made that disallows HTTP(no S) access. The fix here appears to be trivial: I think [this line](https://github.com/richardhenry/hotqueue/blob/bdfcd860342cc4c8fb2ffcfb25ae33ad23bcb5b6/distribute_setup.py#L50) just needs to be `https://` (and a new version would need to be released to PyPI).

Edit: actually, the current code tries to import `setuptools` first, and only if that fails to import does it attempt to access PyPI over HTTP. So, strictly speaking, for my use case (installing w/ `pip`), since `setuptools` is always present, all that is needed is just to release `master` as it is now as a new version. I think it would still be good to change that URL, on the off chance someone runs that code path. (As currently, if that code path runs, it would appear to fail.)

~I'll send a PR for changing the URL.~ Actually, it appears that @ryanstutes has already sent a PR, #20, that would fix this.

**Comment by peterfraedrich** [comment]
PyPi still hasn't picked up this change; the hotqueue package still fails with 403 SSL error when installing from PyPi on Python 2.7

**Comment by richardhenry** [comment]
Sorry this took so long to fix - 0.2.8 is live and this should be fixed now. Please reopen this issue if you're still having trouble installing the library.
